torchci/pages/sli.tsx

```diff
@@ -130,8 +132,9 @@ export function TimeRangePicker({
           <MenuItem value={7}>Last 7 Days</MenuItem>
           <MenuItem value={14}>Last 14 Days</MenuItem>
           <MenuItem value={30}>Last Month</MenuItem>
-          <MenuItem value={90}>Last Quarter</MenuItem>
-          <MenuItem value={180}>Last Half</MenuItem>
+          <MenuItem value={60}>Last 2 Months</MenuItem>
+          <MenuItem value={90}>Last 3 Months</MenuItem>
+          <MenuItem value={180}>Last 6 Months</MenuItem>
```

Comment:
On a similar note, should 6 month be 182 day rather than 180?

Reply:
Same as before, keeping the same standard as in `/cost_analysis`